hedera-node/src/main/resources/bootstrap.properties

```diff
@@ -53,14 +53,14 @@ contracts.allowCreate2=true
 contracts.defaultLifetime=7890000
 contracts.knownBlockHash=
 contracts.localCall.estRetBytes=32
-contracts.maxGas=5000000
+contracts.maxGas=8000000
```

Comment:
Isn't this 4M ? I thought so from this [issue](https://github.com/hashgraph/hedera-fee-tool-js/issues/101)

Reply:
We are (almost certainly) going to remove this entirely, but large factory contracts (e.g. used by SaucerSwap) need 8M gas for certain transactions that do a lot of calls to system contracts.